WcaOnRails/app/views/admin/avatars/index.html.erb

```diff
@@ -4,6 +4,18 @@
   <%= render layout: "nav" do %>
     <h1><%= yield(:title) %></h1>
 
+    <div class="well">
+      <h3>Guidelines</h3>
+      <ul>
+        <li>All avatars should be pictures of real persons.</li>
```

Comment:
The strings in this list seem to have been added to the locale file but they are not used here: is this intentional or overlooked?
(Also guideline 6 is different from the one in the locale)